schema_based_on_diagram.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE patients (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(255) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE invoices(
+  id INT NOT NULL AUTO_INCREMENT,
+  total_amount DECIMAL(10,2) NOT NULL,
+  generated_at DATETIME NOT NULL,
+  payed_at DATETIME,
```

Comment:
- [ ] There is nothing called `DATETIME` as a data type, so Please use `TIMESTAMP` instead or see [this link](https://www.postgresql.org/docs/current/datatype-datetime.html) for more info.